apps/keyboardlegend/src/pages/index.tsx

```diff
@@ -50,7 +51,18 @@ export default function Home() {
               future!
             </h4>
             <div className="max-w-sm mx-auto w-full py-16">
-              <ConvertkitSubscribeForm />
+              <ConvertkitSubscribeForm
+                button={<Button type="submit">Sign Up</Button>}
+                classNames={{
+                  input:
+                    'autofill:caret-black autofill:text-fill-black autofill:bg-white w-full rounded-lg border border-gray-400 focus:ring-2 focus:ring-brand-pink-500 focus:outline-none focus:border-transparent',
+                  label: 'block pb-1',
+                  form: 'space-y-4 w-full',
+                  buttonContainer:
+                    'w-full flex items-center justify-center pt-4',
+                  comment: 'text-sm text-gray-500 text-center pt-6',
+                }}
+              />
```

Comment:
correct styles for keyboardlegend product.

Reply:
with tailwind intellisense plugin you won't get autocomplete in objects like this by default, but you can achieve that by adding this to your vscode settings json: 
```json
"tailwindCSS.experimental.classRegex": [":\\s*?[\"'`]([^\"'`]*).*?"]
```